docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
     environment:
       DATABASE_URL: postgres://postgres@db/askdarcel_development
       TEST_DATABASE_URL: postgres://postgres@db/askdarcel_test
+      ALGOLIA_APPLICATION_ID:
+      ALGOLIA_API_KEY:
```

Comment:
Is there a way we could have people use a shared application id? Ideally, developers wouldn't have to manually configure anything in their environment files.